src/validateAdRequestOptions.ts

```diff
@@ -22,14 +22,14 @@ import {
   isNumber,
   isObject,
   isString,
-  isUndefined,
   isValidUrl,
 } from './common';
+import { RequestOptions } from './types/RequestOptions';
 
-export default function validateAdRequestOptions(options) {
-  const out = {};
+export default function validateAdRequestOptions(options?: RequestOptions) {
+  const out: RequestOptions = {};
 
-  if (isUndefined(options)) {
+  if (typeof options === 'undefined') {
```

Comment:
is this equivalent to 
```suggestion
  if (options === undefined) {
```
:thinking: